### PR TITLE
Register chandrasekhar.is-a.dev

### PR DESCRIPTION
**Pull request by Chandra-Sekhar-Bala**
Register chandrasekhar.is-a.dev with CNAME record pointing to  chandra-sekhar-bala.github.io/Portfolio/
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/https:/chandrasekhar
Comment (CuteDog5695) [inline domains/https:/chandrasekhar:1]: This needs to be a json file mate (also remove the "https:" directory)

File: domains/https:/chandrasekhar [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Chandra-Sekhar-Bala",
+    "email": "sekhar.chandra.5832@gmail.com"
+  },
+  "record": {
+    "URL": "chandra-sekhar-bala.github.io"
```
Comment (CuteDog5695) [inline domains/https:/chandrasekhar:7]:
```suggestion
    "URL": "https://chandra-sekhar-bala.github.io"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/https:/chandrasekhar (hunk 1 above)
Comment (Chandra-Sekhar-Bala) [reply domains/https:/chandrasekhar]: Done!

File: domains/https:/chandrasekhar
Comment (Chandra-Sekhar-Bala) [reply domains/https:/chandrasekhar:1]: Thanks for mentioning, done ✅

File: domains/chandrasekhar
Comment (CuteDog5695) [inline domains/chandrasekhar:1]: you forgot to make the file a json file (just add .json after chandrasekhar)

File: domains/chandrasekhar [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "Chandra-Sekhar-Bala",
+    "email": "sekhar.chandra.5832@gmail.com"
+  },
+  "record": {
+    "URL": "https://chandra-sekhar-bala.github.io"
```
Comment (CuteDog5695) [inline domains/chandrasekhar:7]: Site doesn't seem to be accessible, is the website hosted?

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/chandrasekhar
Comment (Chandra-Sekhar-Bala) [reply domains/chandrasekhar:1]: Oops! 
Done tho 😅

File: domains/chandrasekhar (hunk 2 above)
Comment (Chandra-Sekhar-Bala) [reply domains/chandrasekhar]: Yup! You can check it's working.

**Comment by Chandra-Sekhar-Bala** [comment]
Could you please check @CuteDog5695 ? I have done all the changes.

**Review by iostpa — APPROVED** [review]
Seems fine to me now (also if it doesn't redirect to your website consider using https://redirect.pizza)

**Comment by Chandra-Sekhar-Bala** [comment]
Cool thanks :)

**Pull request merged**